### PR TITLE
fixed minor bugs

### PR DESCRIPTION
**Pull request by SabariGanesh-K**
## Related Issue

Issue #358 

Fixes: #[issue number that will be closed through this PR]
It fixes issue #358 
#### Describe the changes you've made
1)
There was bug in home page which caused it to scroll up automatically after certain seconds.It was basically due to 
=>  Onclick function being sent directly instead of sending it as a event.
=> Another important reason was useeffect hook in navbar page which scrolled the page during render and navbar itself was rendered only after 4 seconds delay caused by settimeout in homepage's useeffect.Both together made the page go up after 4 seconds
Basically issue was not from calculator button / go top button.After solving the above causes , the problem was fixed.

2)
To fix the navbar permanently even after scrolling ,  styles changes were made and most importantly position of navbar was changed to "sticky".

## Checklist:

- [x ] My code follows the style guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [x ] I have commented my code, particularly in hard-to-understand areas.
- [ x] I have made corresponding changes to the documentation.
- [ x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Amit366** [comment]
@SabariGanesh-K why did u add two issues to this PR?
